docs/source/guide/zne-1-intro.myst

```diff
@@ -0,0 +1,144 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use ZNE?
+
+Zero noise extrapolation is an error mitigation technique in which an expectation 
+value is computed at different noise levels and, as a second step, the ideal 
+expectation value is inferred extrapolating the measured results to the zero-noise 
+limit (see the section [What is the theory behind ZNE?](zne-5-theory.myst)).
+
+Both steps — noise scaling and extrapolation — can be applied with Mitiq. The 
+corresponding sub-modules are 
+[`mitiq.zne.scaling`](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.zne.scaling.folding) 
+and 
+[`mitiq.zne.inference`](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.zne.inference).
+
+As with all techniques, ZNE is compatible with any frontend supported by Mitiq:
+
+```{code-cell} ipython3
+import mitiq 
+
+mitiq.SUPPORTED_PROGRAM_TYPES.keys()
+```
+
+In the next cell you can select the frontend used in this tutorial. For example:
+
+```{code-cell} ipython3
+frontend = "cirq"  # Supported: "cirq", "qiskit", "pyquil", "braket", "pennylane".
```

Comment:
Remove comment. Supported frontends are listed above, and will go out of date here.

Reply:
Updated, thanks!